pkg/pool-stable/contracts/ComposableStablePoolProtocolFees.sol

```diff
@@ -75,7 +78,7 @@ abstract contract ComposableStablePoolProtocolFees is
         // swap fees and yield.
         (
             uint256 expectedProtocolOwnershipPercentage,
-            uint256 totalGrowthInvariant
+            uint256 currentInvariantWithLastJoinExitAmp
         ) = _getProtocolPoolOwnershipPercentage(balances, lastJoinExitAmp, lastPostJoinExitInvariant);
 
         // Now that we know what percentage of the Pool's current value the protocol should own, we can compute how
```

Comment:
Same below this comment:

```
        uint256 protocolFeeAmount = ProtocolFees.bptForPoolOwnershipPercentage(
            virtualSupply,
            expectedProtocolOwnershipPercentage
        );
```

Reply:
Sorry could you rephrase?

Reply:
In [L84/L87](https://github.com/balancer-labs/balancer-v2-monorepo/pull/2069/files#diff-06011c15b516ce6b965c6ed91b1b6bc55af19e8ffa3cf37b08ebe07bbfb5b193L84) below this line (you'll need to expand down since that line wasn't changed), the file does 

```
        uint256 protocolFeeAmount = _calculateAdjustedProtocolFeeAmount(
            virtualSupply,
            expectedProtocolOwnershipPercentage
        );
```

This was at some point factored out to a library, like this:

```
        uint256 protocolFeeAmount = ProtocolFees.bptForPoolOwnershipPercentage(
            virtualSupply,
            expectedProtocolOwnershipPercentage
        );
```

The functionality should be the same in any case; it's just a function that was factored out to a library, but I wanted to raise it to double check.